src/pages/FollowUpAnalysis/Form.js

```diff
@@ -19,70 +18,75 @@ const Form = ({
     onStartDateChange,
     endDate,
     onEndDateChange,
-}) => (
-    <>
-        <div className="row">
-            <div className="col-sm-12 col-md-6 col-lg-4">
-                <h3 className={cssPageStyles.formLabel}>
-                    {i18n.t('Data Set')}
-                </h3>
-                <AvailableDataSetsSelect
-                    selected={dataSetIds}
-                    onChange={onDataSetsChange}
-                />
-            </div>
-            <div className="col-sm-12 col-md-6 col-lg-4">
-                <h3 className={cssPageStyles.formLabel}>
-                    {i18n.t('Parent organisation unit')}
-                </h3>
-                <AvailableOrganisationUnitsTree
-                    onChange={onOrganisationUnitChange}
-                />
-            </div>
-            <div className="col-sm-12 col-md-6 col-lg-4">
-                <DatePicker
-                    id="start-date"
-                    textFieldStyle={jsPageStyles.inputForm}
-                    floatingLabelText={i18n.t('Start Date')}
-                    onChange={onStartDateChange}
-                    defaultDate={new Date()}
-                    maxDate={endDate}
-                    value={startDate}
-                />
-                <DatePicker
-                    id="end-date"
-                    textFieldStyle={jsPageStyles.inputForm}
-                    floatingLabelText={i18n.t('End Date')}
-                    onChange={onEndDateChange}
-                    defaultDate={new Date()}
-                    minDate={startDate}
-                    value={endDate}
-                />
+}) => {
+    const { systemInfo = {} } = useConfig()
+    const { calendar = 'gregory' } = systemInfo
+
+    return (
+        <>
+            <div className="row">
+                <div className="col-sm-12 col-md-6 col-lg-4">
+                    <h3 className={cssPageStyles.formLabel}>
+                        {i18n.t('Data Set')}
+                    </h3>
+                    <AvailableDataSetsSelect
+                        selected={dataSetIds}
+                        onChange={onDataSetsChange}
+                    />
+                </div>
+                <div className="col-sm-12 col-md-6 col-lg-4">
+                    <h3 className={cssPageStyles.formLabel}>
+                        {i18n.t('Parent organisation unit')}
+                    </h3>
+                    <AvailableOrganisationUnitsTree
+                        onChange={onOrganisationUnitChange}
+                    />
+                </div>
+                <div className="col-sm-12 col-md-6 col-lg-4">
+                    <div className={cssPageStyles.datepickers}>
+                        <CalendarInput
+                            onDateSelect={onStartDateChange}
+                            date={startDate}
+                            calendar={calendar}
+                            locale="en"
```

Comment:
I do think it would be good to do so, but have some concerns about mismatches between (non-standard) locale codes used in DHIS2 and the locale codes recognised by the browser. I would say there are two possible ways to go here:
1. If the calendar hooks are smart enough to fallback to some default locale if the provided locale is not recognised by the browser, then we should pass the user locale to the `CalendarInput`
2. If not, then let's just keep passing `en` to prevent the component being useless

I definitely remember @kabaros and myself discussing the fallback strategy described in point 1, but I can't remember if the current version already has support for this. Since @kabaros mentioned that _"we established the pattern in analytics already"_, it seems we already do have support and we should go with option 1.